packages/react/src/experimental/Forms/EntitySelect/Content/index.tsx

```diff
@@ -57,6 +59,8 @@ export const Content = ({
   disabled?: boolean
   hiddenAvatar?: boolean
   actions?: Action[]
+  onCreate?: (partialName: string) => void
+  onCreateLabel?: string
```

Comment:
Two thoughts here:
- those two props will be set together in most of the cases. It could be better to either fuse those two props into one or split the type definition and make the impossible state (only one of those two is set) impossible
- is creation the only way to use this top item? shouldn't we use more generic prop to keep the usage open?